spec/definitions/boxes/definitions/base_ubuntu14.04_kvm/definition.rb

```diff
@@ -0,0 +1,45 @@
+Veewee::Session.declare({
+  cpu_count: "1",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.